### DISCUSSION TITLE
High compute usage by GraphQL methods behind the Scenes.

### DISCUSSION THREAD

**Discussion by narayan-meesho** [graphql-java-kickstart/graphql-spring-boot#952] [Q&A]
Hi,
We are using graphQL for a very high scale service ( JAVA ) , on doing a detailed analysis of compute used by different layers of our service, we observed that its kickstart graphQL library methods using most of the compute around (37- 40 % ) of cpu usage of a machine. Wanted to understand if any other reports of same high cpu usage by graphql is made by other folks and like what is expected in terms of cpu compute used by GraphQL behind the scenes in invoking FieldResolvers. I would request the developers to help here, I am ready to share more details if required.

**Comment by narayan-meesho** [comment]
@oliemansm bumping this up, appreciate your response here.

**Reply by oliemansm** [reply]
You're bumping this up because you didn't get a response within five hours?

**Reply by oliemansm** [reply]
This lib is a starter wrapping graphs-Java which does the actual heavy lifting. You could try using just plain GraphQL-Java to see if you can reproduce the problem. Because my first suspect would be that library. If you can reproduce it just using that lib then raise the issue at their project. If you can only reproduce it icw this project then we'd need a code sample in a repo for example demonstrating the issue.

**Reply by narayan-meesho** [reply]
No, was not expecting the response in five hours, only to highlight it to some core team member. Thanks for your response, let me check with the core library once and revert here.
